pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -704,20 +704,23 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 	podResources := cm.ResourceConfigForPod(pod, enforceCPULimits, uint64((m.cpuCFSQuotaPeriod.Duration)/time.Microsecond), false)
 	if podResources == nil {
 		klog.ErrorS(nil, "Unable to get resource configuration", "pod", klog.KObj(pod))
-		result.Fail(fmt.Errorf("unable to get resource configuration processing resize for pod %s", pod.Name))
-		return
+		err := fmt.Errorf("unable to get resource configuration processing resize for pod %s", pod.Name)
+		result.Fail(err)
```

Comment:
For my alternative suggestion, we should change these failure results to a new sync result, with a new `ResizePod` action.

Reply:
thanks!! much cleaner :)

I did have to modify the loop in kubelet that checks for syncErrors a bit though, to make sure it always checks for resize errors